apps/permissions.py

```diff
@@ -26,6 +28,10 @@
 
 PROFILES_WALLET_URL = f'{settings.PROFILES_URL}/api/v1/wallet'
 
+LOG = logging.getLogger('transmission')
+
+NON_OWNER_ACCESS_METHODS = ('GET', 'PATCH', )
```

Comment:
ConfigurableGenericViewSet also supports configuring per-action permission classes.

Reply:
@ClovisDj please use the support in ConfigurableGenericViewSet for this